LibreNMS/Validations/Database.php

```diff
@@ -328,8 +328,8 @@ private function checkSchema(Validator $validator)
         if (empty($schema_update)) {
             $validator->ok('Database schema correct');
         } else {
-            $result = ValidationResult::fail('We have detected that your database schema may be wrong, please report the following to us on Discord (https://t.libren.ms/discord) or the community site (https://t.libren.ms/5gscd):')
-                ->setFix('Run the following SQL statements to fix.')
```

Comment:
How about add on most distros you can access mysql with `mysql -u root librenms`?

Reply:
I was thinking of updating it later so it's copy-paste `lnms tinker --execute` commands

Reply:
that way the database connection/selection is already included.
We could just add a button from the webui, but feels a bit... hmm.. prone to breakage